bcda/models/models.go

```diff
@@ -1,19 +1,28 @@
 package models
 
 import (
-	"github.com/CMSgov/bcda-app/bcda/auth"
+	"crypto/rsa"
+	"fmt"
 	"github.com/CMSgov/bcda-app/bcda/database"
+	"github.com/CMSgov/bcda-app/bcda/secutils"
 	"github.com/jinzhu/gorm"
 	"github.com/pborman/uuid"
+	"os"
 )
 
 func InitializeGormModels() *gorm.DB {
+	fmt.Print("Intitialize bcda models")
 	db := database.GetGORMDbConnection()
 	defer db.Close()
 
 	// Migrate the schema
 	// Add your new models here
+	// This should probably not be called in production
```

Comment:
this is fine to run in production.  it only adds columns and doesn't delete any data.